src/IVSPlayer.tsx

```diff
@@ -185,15 +184,8 @@ const IVSPlayerContainer = React.forwardRef<IVSPlayerRef, Props>(
     }, []);
 
     useEffect(() => {
-      if (initialized.current || autoplay) {
```

Comment:
We should retain the `if (initialized.current) {` check here so that we do not trigger a pause / play on the initial component creation.